docs/user-guide/getting-started.md

```
@@ -22,6 +22,15 @@ brew install EvilBit-Labs/tap/opndossier
 go install github.com/EvilBit-Labs/opnDossier@latest
 ```
 
+!!! warning "`go install` names the binary `opnDossier`"
+    `go install` takes the binary name from the module path, so it installs `opnDossier` (capital D), not `opndossier`. On Linux and other case-sensitive filesystems the commands in this guide will not resolve until you rename it:
+
+    ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)